### PR TITLE
Bug fixes & improvements

### PR DESCRIPTION
**Pull request by sassypocoyo**
Fixed marker layout bug on the Location list where the marker png would be resized depending on the amount of text on the authors list. Also added the dynamic author icons to Location.

### CODE REVIEW

File: lib/utils.tsx [hunk 1]
```diff
@@ -68,3 +69,10 @@ export const sortByOldest = (a: IPin, b: IPin) => {
     DateTime.fromISO(a.date).toMillis() - DateTime.fromISO(b.date).toMillis()
   );
 };
+
+export function getAuthorIcon(author: string | string[]) {
```
Comment (nelsonmestevao) [inline lib/utils.tsx:73]: This should be a component because it it JSX 🙌

**Review by nelsonmestevao — DISMISSED** [review]

File: lib/utils.tsx (hunk 1 above)
Comment (sassypocoyo) [reply lib/utils.tsx]: Done :rocket:

File: lib/utils.tsx [hunk 2]
```diff
@@ -68,3 +68,16 @@ export const sortByOldest = (a: IPin, b: IPin) => {
...
   );
 };
+
+// <AuthorIcon/> - Used on Marker and Location components
+
+interface Author {
+  author: string | string[];
+}
+
+export const AuthorIcon = ({ author }: Author) => {
```
Comment (nelsonmestevao) [inline lib/utils.tsx:78]: You need to move this to the components folder and leave the utils.ts for only functions 💪

**Review by nelsonmestevao — DISMISSED** [review]

File: lib/utils.tsx [hunk 3]
```diff
@@ -68,3 +68,16 @@ export const sortByOldest = (a: IPin, b: IPin) => {
     DateTime.fromISO(a.date).toMillis() - DateTime.fromISO(b.date).toMillis()
   );
 };
+
+// <AuthorIcon/> - Used on Marker and Location components
+
+interface Author {
```
Comment (nelsonmestevao) [inline lib/utils.tsx:74]: Call this AuthorIconProps 💪

File: lib/utils.tsx (hunk 2 above)
Comment (sassypocoyo) [reply lib/utils.tsx]: Done :+1:

**Comment by sassypocoyo** [comment]
@nelsonmestevao can you approve this now?

**Review by nelsonmestevao — APPROVED** [review]

**Pull request merged**